setup.cfg

```diff
@@ -18,15 +18,18 @@ search = __version__ = "{current_version}"
 replace = __version__ = "{new_version}"
 
 [flake8]
-exclude = 
+exclude =
 	.git,
 	docs,
 	build,
 	.eggs,
 	tests
 max-line-length = 88
 max-complexity = 12
-ignore = 
+ignore =
```

Comment:
IMHO it is a good practice to comment which and why rules are ignored.

Reply:
Sure thing!